cuebot/src/main/java/com/imageworks/spcue/dispatcher/HostReportHandler.java

```diff
@@ -80,6 +84,14 @@ public class HostReportHandler {
     private JobManagerSupport jobManagerSupport;
     private JobDao jobDao;
     private LayerDao layerDao;
+    @Autowired
+    private Environment env;
+    @Autowired
+    private CommentManager commentManager;
+    // Comment constants
+    private static final String SUBJECT_COMMENT_FULL_MCP_DIR = "Host set to REPAIR for not having enough storage " +
+            "space on /mcp";
```

Comment:
It's my understanding that for most users, the thing we call "MCP" will not actually be the `/mcp` directory:

https://github.com/AcademySoftwareFoundation/OpenCue/blob/6703e07de6982c7372647d2aa0216b86979b320a/rqd/rqd/rqmachine.py#L519-L523

Ideally we would update all of OpenCue to use more fitting terminology, but that's a big project and not something we need to do right now.

However, in things like user-readable strings, and code comments, is there a different term we can use? This line is one example but there are a few others in this PR.

The current string would seem to be a bit misleading, if users have RQD hosts that aren't actually using the literal `/mcp` directory.

Reply:
Hi @bcipriano 

I propose we use the term ```TempDir``` to replace ```Mcp```. What do you think?

I will replace all the ```Mcp``` by ```TempDir``` in my pull resquest. 

However, as you said, we can create a new task later on to update all of OpenCue code to fit the terminology.

Reply:
Done!

I changed the comments, variables and methods reference from "mcp" to "temporary directory" or "TempDir".

See commit: https://github.com/AcademySoftwareFoundation/OpenCue/pull/1306/commits/04ec1068a80791f356d3517300eb1bb02f956ddb